wtpython/display.py

```diff
@@ -126,17 +126,15 @@ async def action_next_question(self) -> None:
         if len(self.data["results"]) > self.index + 1:
             self.index += 1
             await self.body.update(self.create_body_text())
-            self.body.y = 0
-            self.body.target_y = 0
+            self.body.focus()
```

Comment:
these were there so that when switching between questions, the scrollbar would move to the top. otherwise the scrollbar remains at the position it was in the previous view

an alternative is to use `await self.body.key_home()` but that animates the scrollbar which I didn't really like